<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionpreviewWindow</class>
 <widget class="QDialog" name="ActionpreviewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>490</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>동작 편집</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: #EFEFEF;
}</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QFrame" name="frame_main">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>340</width>
     <height>470</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
	background: white;
	border:1px solid black;
	border-radius: 5px;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="button_save">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>430</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
      <hintingpreference>PreferVerticalHinting</hintingpreference>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #EFEFEF;
    padding: 2px 4px;
}</string>
    </property>
    <property name="text">
     <string>저장</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::InsertText"/>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_cancel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>430</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
      <hintingpreference>PreferVerticalHinting</hintingpreference>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #EFEFEF;
    padding: 2px 4px;
}</string>
    </property>
    <property name="text">
     <string>취소</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::InsertText"/>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_preview">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black;
	background-color: #EAEFEF;
	border: 1px solid black;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	color:black;
	border: 1px solid black;
	border-radius: 5px; 
	padding-left:10px;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="placeholderText">
     <string>동작의 이름을 입력하세요</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>300</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	background: white;
	border:1px solid black;
	border-radius: 5px;
}
</string>
    </property>
    <property name="title">
     <string>클릭 영역 좌표</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_Y">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>35</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	color:black;
	border: 1px solid black;
	border-radius: 5px; 
	padding-left:10px;
}</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_X">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>35</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	color:black;
	border: 1px solid black;
	border-radius: 5px; 
	padding-left:10px;
}</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_Y">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>35</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
color:black;</string>
     </property>
     <property name="text">
      <string>Y :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_X">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
color:black;</string>
     </property>
     <property name="text">
      <string>X :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox_time">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string> 시간 조정 관련 동작일 경우 체크하세요</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
